src/java.net.http/share/classes/jdk/internal/net/http/common/DebugLogger.java

```diff
@@ -41,10 +41,72 @@
  * format string. MessageFormat-like formatting is enabled by the presence of
  * "{0" or "{1".
  *
- * <p> See {@link Utils#getDebugLogger(Supplier, boolean)} and
- * {@link Utils#getHpackLogger(Supplier, boolean)}.
+ * <p> See {@link Utils#getDebugLogger(Supplier)} and
+ * {@link Utils#getHpackLogger(Supplier)}.
  */
 final class DebugLogger implements Logger {
+
+    /**
+     * A DebugLogger configuration is composed of three levels.
+     * The three levels can be configured independently, but are
+     * typically either Level.ALL or Level.OFF.
+     *
+     * @param outLevel the level above which messages will be directly
+     *                printed to {@link System#out}
+     * @param errLevel the level above which messages will be directly
+     *                printed to {@link System#err}
+     * @param logLevel the level above which messages will be forwarded
+     *               to an underlying {@link System.Logger}
```

Comment:
Hello Daniel, looking at the code, it appears to me that the documentation of all these three levels should say "the level above or equal to which ..." (i.e. `>=` instead of `>`).

Reply:
I noticed that certain other parts in this class use the term "at or above". I think we should use the same here.